metaci/fixtures/factories.py

```diff
@@ -46,7 +46,7 @@ def fake_name(prefix=None):
     the class."""
     return factory.LazyAttribute(
         lambda a: (getattr(a, "name_prefix", None) or prefix or "")
-        + factory.Faker("word").generate({})
+        + factory.Faker("word").generate({"locale": None})
```

Comment:
Adding locale here seems to be causing problems when the tests run on Heroku CI. What was the error you were trying to fix?

Reply:
Couldn't seem tot replicate. Removed it.